### PR TITLE
refactor: made all packages to use shared eslint/prettier config

### LINKED ISSUE #224: Make all packages use shared eslint/prettier config

### PR DESCRIPTION
**Pull request by Chiadikaobixo**
fix #224

### Description
Elaborate on the subject, motivation, and context.

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: apps/backoffice/.eslintrc.cjs [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
-    'prettier',
-    'react-app',
-    'react-app/jest',
-  ],
+  ...config,
+  extends: [...config.extends, 'react-app', 'react-app/jest'],
```
[the remark this answers is not in the document]
Comment (Chiadikaobixo) [reply apps/backoffice/.eslintrc.cjs]: Thanks for the corrections, I have fix that.

File: services/workflows-service/.eslintrc.cjs [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
-    'plugin:import/recommended',
-    'plugin:import/typescript',
-    'prettier',
-  ],
+  ...config,
+  extends: [...config.extends, 'plugin:import/recommended', 'plugin:import/typescript'],
```
[the remark this answers is not in the document]
Comment (Chiadikaobixo) [reply services/workflows-service/.eslintrc.cjs]: Thanks for the corrections, I have fix that.

**Review by alonp99 — APPROVED** [review]

**Review by alonp99 — APPROVED** [review]

**Comment by alonp99** [comment]
Thanks @Chiadikaobixo , this looks good

Can you resolve the conflicts so I can merge this?

**Comment by alonp99** [comment]
Thanks @Chiadikaobixo , this looks good

Can you resolve the conflicts so I can merge this?

**Pull request closed without merging**